people/compensation.md

```diff
@@ -64,5 +63,59 @@ We have [internal salary documents for the specific positions and levels](https:
 
 ## Benefits
 
-Benefits are offered by {term}`Code for Science and Society`.
-They are offered via [TriNet](https://www.trinet.com/) as described in [the CS&S employee handbook](https://drive.google.com/file/d/1anHo8P09gjGLnUfGj2ceSDxvJTYwMeS1/view?usp=sharing).
+This section describes our benefits policy across all team members at 2i2c. 2i2c team members include people who are designated as employees and as contractors, depending on their role at 2i2c.
+
+Benefits are available to employees only and are managed through {term}`Code for Science and Society`’s relationship with a service partner or PEO. For example, employees who are US based and employed through {term}`Code for Science and Society`’s relationship with [Trinet](https://www.trinet.com/), will have access to health coverage and retirement benefits through Trinet. Employees based outside the US and employed through a relationship with a PEO like Remote will have access to benefits available in their region through that PEO. More details can be found at the [the CS&S employee handbook](https://drive.google.com/file/d/1anHo8P09gjGLnUfGj2ceSDxvJTYwMeS1/view?usp=sharing).
```

Comment:
Can we group these benefits into a category of some sort? maybe payroll? (no idea if this is correct)

The reasoning is that we have this big section called _**"Benefits"**_ that describes the overall policy for employees & contractors. 

But then, there's the following sentence _"Benefits are available to employees only"_ which makes it a bit unclear that the extras defined in their own section below are available to everyone and not to employees only.

If we'd find a category for this, let's call it _payroll_ (again, I'm not sure this is correct), something like _"Payroll benefits are available to employees only"_ would be more clear that it refers to its own category of benefits.

What do you think?

Reply:
I restructure things a little bit more in my [latest commit](https://github.com/2i2c-org/team-compass/pull/682/commits/1cf40ee0d209e80db3e7c416e93a21b9da85ead9), let me know if that makes more sense to you!

Reply:
Resolving for now, but feel free to unresolve if you disagree.